src/cascadia/TerminalSettingsModel/AIConfig.cpp

```diff
@@ -162,7 +162,7 @@ winrt::hstring AIConfig::_RetrieveCredential(const wil::zwstring_view credential
     }
     catch (...)
     {
-        return L"";
+        return winrt::hstring{};

```

Comment:
for the prior nit, this is an excellent case - `return {}` works because the compiler already knows the return tupe

Reply:
(resolving for now but will keep in mind if I touch that part of the code, same as the other one)